### PR TITLE
fix: change to download package lists synchronously

### LINKED ISSUE #746: Fail to update the package list
## Describe the bug

It is failing to update the package list.

## To Reproduce

1. Press the "更新" button which checks package data.

## Expected behavior

Update all package lists.

## Desktop (please complete the following information):

- OS:  Windows 11
- Version:  21H2

### PR DESCRIPTION
**Pull request by hal-shu-sato**
## Description

- Change to download package lists synchronously.

## Related Issue

- Fix #746

## Motivation and Context

Refer to #746 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by mitosagi** [comment]
It will be a few days before I can review it as I do not have the environment to debug it.

File: src/renderer/main/packageUtil.ts [hunk 1]
```diff
@@ -144,14 +144,12 @@ async function getPackages(packageDataUrls: string[]) {
...
 async function downloadRepository(packageDataUrls: string[]) {
-  await Promise.all(
-    packageDataUrls.map((packageRepository) =>
-      download(packageRepository, {
-        subDir: 'package',
-        keyText: packageRepository,
-      })
-    )
-  );
+  for (const packageRepository of packageDataUrls) {
```
Comment (yumetodo) [inline src/renderer/main/packageUtil.ts:147]: Why don't you add comment why we download synchronously?

Comment (hal-shu-sato) [reply src/renderer/main/packageUtil.ts:148]: I added about it.

**Review by mitosagi — APPROVED** [review]
I have tried 5 clean installs and they all work fine.
Good fix.

**Pull request merged**